tests/README.md

```diff
@@ -1,13 +1,33 @@
-# Running Vanilla's Tests
+# Vanilla's Tests
 
 Vanilla's tests are designed to work on [Travis CI](https://travis-ci.org/), but you can set your local environment up
-to run the tests with minimal setup. Here is what you need.
+to run the tests with minimal setup. 
 
-1. Your localhost MySQL server will need a user named "travis" with a blank password. This user will need to be able to
-create and drop databases. The only database that the tests used is called "vanilla_test".
+## Requirements
 
-2. The copy of Vanilla you have installed, must respond to http://vanilla.test:8080. You can use the NGINX template in
-tests/travis/nginx/default-site.tpl.conf as a guideline. If you are on Apache then the default .htaccess file should
-work for you.
+1. Your localhost MySQL server must have a user named `travis` with a blank password and permission to
+create and drop databases. The only database that the tests use is `vanilla_test`.
 
-3. Remember to install all of the developer dependencies with `composer install`.
\ No newline at end of file
+2. Your copy of Vanilla must respond to `http://vanilla.test:8080`. You can use the nginx template in
+`tests/travis/nginx/default-site.tpl.conf` as a guideline. If you are on Apache, the default `.htaccess` file should work for you.
+
+3. All of the developer dependencies installed with `composer install`.
+
+4. [PHPUnit installed](https://github.com/sebastianbergmann/phpunit#installation).
+
+## Running
+
+1. Go to your Vanilla install directory.
+2. `phpunit -c phpunit.xml.dist --testsuite Library`
+
+To run without the `--testsuite` flag requires that `/cgi-bin` be mapped, as in the template mentioned above. In nginx, try:
+
+```
+    location ~* "^/cgi-bin/.+\.php(/|$)" {
```

Comment:
Don't indent. This is also something that is in the default-site.tpl.conf. I really don't want people that can't understand the nginx template file to think they can just copy and past snippets like this. What if their file structure is different for example?